src/store/modules/wallets/wallets-plugin.ts

```diff
@@ -0,0 +1,14 @@
+import VuexPersistence from 'vuex-persist'
+import { CoinSymbol, PluginWalletsState } from '@/store/modules/wallets/types'
+
+const walletsPersistencePlugin = new VuexPersistence({
+  key: 'adm-wallets',
+  storage: window.localStorage,
+  reducer: (state: PluginWalletsState) => {
```

Comment:
```suggestion
  reducer: (state: RootState) => {
```

You don't need an additional type. The argument of the reducer is actually the root state. Remove also the `type PluginWalletsState` declaration if you are not using it in other places.